client/pages/member-edit-modal/modal.template.html.slim

```diff
@@ -5,7 +5,7 @@ form novalidate=true name='membershipForm' ng-submit='membershipForm.$valid && s
     h3.modal-title ng-if='!editedMembership.id' New Member
   .modal-body
     .form-group ng-if='!membership.id'
-      label Type
+      label-required for='technical' Type
```

Comment:
I wouldn't put "required" next to a radio button that cannot be unselected anyway.